csrc/scheduler/vectorize_helper.cpp

```diff
@@ -1201,13 +1201,29 @@ int64_t getVectorizationSize(
   return vectorize_size;
 }
 
-size_t getExpandedVectorization(
```

Comment:
A big portion of `getVectorizationFactor` has been removed, so I just cut-paste the function body of this function into `getVectorizationFactor` and remove this function.